packages/itwin/property-grid/src/components/Header.tsx

```diff
@@ -45,7 +45,7 @@ function DebouncedSearchBox({ onSearchStringChange, onClose, onOpen, className }
 
     const timeoutId = setTimeout(() => {
       onSearchStringChangeRef.current(inputValue);
-    }, 25);
+    }, 500);
```

Comment:
While 25 is maybe too little, 500 seems a bit too much - why did you make the change?

Reply:
Because when it was set to 25 the `filter-properties` feature would be constantly reported while typing, so it seemed too low. Switched to 500 because that was the debounce timeout set in `UniqueValuesSelector`. What value should I use instead?

Reply:
The difference is that unique values selector may have to make an RPC request to get the filtered values values, while the property grid already has everything and can filter synchronously. Could we instead debounce reporting of the feature, e.g. if you filter 5 times in a timeframe of 1 second, report just one usage.